src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -7227,6 +7486,160 @@ Label90: ;
 		return ACH;
 	}
 
+	void OccupantVentilationControlProp::calc(
+		int const ZoneNum,
+		int const SurfNum,
+		int const PrevOpeningstatus,
+		Real64 const TimeOpenDuration,
+		Real64 const TimeCloseDuration,
+		int & OpeningStatus,
+		int & OpeningProbStatus,
+		int & ClosingProbStatus
+		)
+	{
+		using DataHeatBalance::MRT;
+
+		Real64 Tcomfort; // Thermal comfort temperature
+		Real64 ComfortBand; // Thermal comfort band
+		Real64 Toperative; // Oprative temperature
+		Real64 OutDryBulb;
+
+		// flow
+
+//		if ( PrevOpeningstatus == 1 ) {
+		if ( TimeOpenDuration > 0 ) {
+			if ( TimeOpenDuration >= MinOpeningTime ) {
+				OpeningStatus = 0; // free operation
```

Comment:
@lgu1234 Could we think about making OpenStatus, OpeningProbStatus, and ClosingProbStatus into enumerations rather than using integers? I'm not sure if there is a way to do it with one, but it will be easier to understand what is going on with names (ForceOpen, etc.) rather than values.